frontend/public/scss/home-page/home-page-video-component.scss

```diff
@@ -70,49 +73,33 @@
     }
 
     .home-page-video-wrapper {
-      max-width: 900px;
-      width: 64%;
-      height: 500px;
-      @include media-breakpoint-down(lg) {
+      padding: 0;
+      .home-page-video-player {
         width: 100%;
         max-width: 100%;
```

Comment:
I don't think max-width needs to be defined if width is 100%

Reply:
I had a lot of problems getting it to behave. You're right... but I think I had it in there as "please, please behave"